core/src/container.cpp

```diff
@@ -365,8 +365,13 @@ void ContainerBase::insert(Stage::pointer&& stage, int before) {
 	if (!stage)
 		throw std::runtime_error(name() + ": received invalid stage pointer");
 
+	if (stage->trajectoryExecutionInfo().controller_names.empty()) {
+		stage->setTrajectoryExecutionInfo(this->trajectoryExecutionInfo());
+	}
```

Comment:
This is not the correct way of inheriting properties from parent to child (I think this is intended here).
Users should make this inheritance explicit _in their code_ via:
`Property::configureInitFrom(Stage::PARENT, "trajectory_execution_info");`

Reply:
The intend here is to make inheriting TrajectoryExecutionInfo from the parent in the container default except, the stage has explicitly configured it's own trajectory execution info. If a user has already done this via configureInitFrom, nothing happens here. Do you think that is a bad default?

Reply:
Yes, I do consider this a bad default, because it contradicts the philosophy of _explicit_ property inheritance, which @v4hn requested from the very beginning. We should not deviate from that philosophy. Please revert that part.

Reply:
https://github.com/ros-planning/moveit_task_constructor/pull/502